python/cudf/bindings/cudf_cpp.pyx

```diff
@@ -50,11 +49,29 @@ gdf_dtypes = {
     GDF_STRING:            np.object_,
 }
 
+np_pa_dtypes = {
+    np.float64:     pa.float64(),
+    np.float32:     pa.float32(),
+    np.int64:       pa.int64(),
+    np.int32:       pa.int32(),
+    np.int16:       pa.int16(),
+    np.int8:        pa.int8(),
+    np.bool_:       pa.int8(),
+    np.datetime64:  pa.date64(),
+    np.object_:     pa.string(),
+    np.str_:        pa.string(),
+}
+
 def gdf_to_np_dtype(dtype):
     """Util to convert gdf dtype to numpy dtype.
     """
     return np.dtype(gdf_dtypes[dtype])
 
+def np_to_pa_dtype(dtype):
```

Comment:
I wonder if this file is the best location for this utilities to convert dtypes

Reply:
I had thoughts about this too, but since it lived in `_gdf.py` I wanted to eliminate that file. I'm likely going to take another pass over these after it's all converted to Cython and will possibly move this out into `utils` or somewhere else then.